core/sail/fts/elasticsearch/src/main/java/org/eclipse/rdf4j/sail/elasticsearch/ElasticsearchIndex.java

```diff
@@ -173,7 +174,7 @@ public void initialize(Properties parameters)
 		throws Exception
 	{
 		super.initialize(parameters);
-		indexName = parameters.getProperty(INDEX_NAME_KEY, DEFAULT_INDEX_NAME);
+		indexName = parameters.getProperty(INDEX_NAME_KEY, UUID.randomUUID().toString());
```

Comment:
Don't agree this should be set to some random value, especially if we know that uuid might cause filename length issues. The default index name should be fine.